ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -149,6 +150,23 @@ protected void init(Context context, AttributeSet attrs, int defStyleAttr, int d
             textPaint.setShadowLayer(styledAttributes.shadowRadius, styledAttributes.shadowDx,
                     styledAttributes.shadowDy, styledAttributes.shadowColor);
         }
+        if (styledAttributes.textStyle != 0) {
+            final int textStyle = styledAttributes.textStyle;
+            int flags = textPaint.getFlags();
+            if ((textStyle & AssistStructure.ViewNode.TEXT_STYLE_BOLD) != 0) {
```

Comment:
yeah I was looking at the wrong textview source code, should use constants from `Typeface`.